qaframework-bdd-tests/src/test/resources/features/refapp-2.x/provider/provider.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Provider Management
+
+  Background: 
+    Given a user clicks on the administration app
+    Then system loads administration page
+    And a user clicks on the manage provider link
+    Then system loads manage provider page
+
+  @selenium
+  @provider
+  Scenario: Adding Provider
+    When a user clicks on the add provider link
+    Then system loads add provider page
+    And a user fills the provider form
+    When a user searches for the created provider
+    And a user clicks on the provider in the search results
+    Then a user deletes provider forever
```

Comment:
is deleting a provider part of the `Scenario: Adding Provider`?

Reply:
unfortunately it is part of the [AddProviderTest.java Legacy Test](https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-legacy-tests/src/test/java/org/openmrs/contrib/qaframework/legacy/AddProviderTest.java)

Reply:
or does the legacy test need a fix?

Reply:
> or does the legacy test need a fix?

Such flaws need to be fixed.

Reply:
oh alright , so do i add that fix on this pull request?